src/modules/hip/hip_tensor_statistical_operations.hpp

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2019 - 2022 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
update date, applies to all such comments in other files

Reply:
Done